src/test/kotlin/com/timcastelijns/room15bot/usecases/AhmadUseCaseTest.kt

```diff
@@ -0,0 +1,156 @@
+            "I dunno man, I'm just an intern @ Blizzard Entertainment.",
+            "I dunno man, I'm just an intern @ LinkedIn.",
+            "I dunno man, I'm just an intern @ Oracle.",
+            "I dunno man, I'm just an intern @ Tata Consultancy Services.",
+            "I dunno man, I'm just an intern @ Slack Technologies, Inc..",
+            "I dunno man, I'm just an intern @ Mailchimp.",
+            "I dunno man, I'm just an intern @ Namecheap.",
+            "I dunno man, I'm just an intern @ Square.",
+            "I dunno man, I'm just an intern @ TeamViewer.",
+            "I dunno man, I'm just an intern @ Atlassian.",
+            "I dunno man, I'm just an intern @ Zendesk.",
+            "I dunno man, I'm just an intern @ MuleSoft.",
+            "I dunno man, I'm just an intern @ Squarespace.",
+            "I dunno man, I'm just an intern @ MathWorks.",
+            "I dunno man, I'm just an intern @ LiveChat.",
+            "I dunno man, I'm just an intern @ Kahoot!.",
+            "I dunno man, I'm just an intern @ Deputy.",
+            "I dunno man, I'm just an intern @ Telegram Messenger.",
+            "I dunno man, I'm just an intern @ GitHub.",
+            "I dunno man, I'm just an intern @ JetBrains.",
+            "I dunno man, I'm just an intern @ Balsamiq.",
+            "I dunno man, I'm just an intern @ CircleCI.",
+            "I dunno man, I'm just an intern @ Wondershare.",
+            "I dunno man, I'm just an intern @ Acronis.",
+            "I dunno man, I'm just an intern @ Notepad++.",
+            "I dunno man, I'm just an intern @ Trimble.",
+            "I dunno man, I'm just an intern @ PayScale.",
+            "I dunno man, I'm just an intern @ Stack Exchange.",
+            "I dunno man, I'm just an intern @ Reddit.",
+            "I dunno man, I'm just an intern @ Foxconn.",
+            "I dunno man, I'm just an intern @ Alphabet Inc..",
+            "I dunno man, I'm just an intern @ Huawei.",
+            "I dunno man, I'm just an intern @ Hitachi.",
+            "I dunno man, I'm just an intern @ Panasonic.",
+            "I dunno man, I'm just an intern @ Accenture.",
+            "I dunno man, I'm just an intern @ Taiwan Semiconductor Manufacturing.",
+            "I dunno man, I'm just an intern @ Texas Instruments.",
+            "I dunno man, I'm just an intern @ Qualcomm.",
+            "I dunno man, I'm just an intern @ Micron Technology.",
+            "I dunno man, I'm just an intern @ Nintendo.",
+            "I dunno man, I'm just an intern @ Broadcom.",
+            "I dunno man, I'm just an intern @ ASML.",
+            "I dunno man, I'm just an intern @ Applied Materials.",
+            "I dunno man, I'm just an intern @ Salesforce.",
+            "I dunno man, I'm just an intern @ Raspberry Pi Foundation.")
+
+    @Test
+    fun testRandomAhmadReply() {
+        assertTrue { internships.contains(AhmadUseCase().execute(Unit)) }
```

Comment:
Better use a loop here

Reply:
Really? The messages are returned at random, so even if I loop on it I don't think it'll catch all the mesages. Or are you saying it like "loop 5 times"?